styles.css

```diff
@@ -111,13 +128,68 @@ ul li {
   display: inline-block;
 }
 
+@media only screen and (min-width: 768px) {
+  .navbar {
+    padding: 2rem 2rem;
+  }
+
+  .my-logo-wrapper {
+    margin-left: 8rem;
+  }
+
+  .hamburger-menu {
+    display: none;
+  }
+
+  .menu-style {
+    display: block;
+  }
+
+  .headline {
+    justify-content: flex-start;
+  }
+
+  .headline-social-media-links {
+    flex-direction: column;
+    position: absolute;
+    bottom: 20rem;
+    left: 1.8rem;
+  }
+
+  .headline-link-img {
+    margin-bottom: 1.3rem;
+  }
+
+  .headline-wrapper {
+    width: 100%;
+    padding-left: 10em;
+  }
+
+  .greeting-title {
+    font-size: 6.2em;
+  }
+
+  .greeting-description1 {
+    font-size: 3.2em;
+  }
+
+  .greeting-description2 {
+    font-size: 2em;
+  }
+
+  .illustration {
+    background-position: 4.5rem -7.5rem;
+    background-size: cover;
+  }
+}
+
 .works-title {
   font-family: "Crete Round", serif;
   font-weight: 400;
   color: #1c1a19;
   font-size: 40px;
   text-align: center;
-  margin-top: 100px;
+  margin-top: 5rem;
 }
```

Comment:
- [ ] To match the Figma design and for the project requirements, kindly make sure you also use the correct text color to your `Works` title i.e. this `color: #091e42;` instead of `color: #1c1a19;`